flask/build.gradle

```diff
@@ -0,0 +1,57 @@
+plugins {
+    id 'groovy'
+    id "java-gradle-plugin"
+}
+
+evaluationDependsOn("common")
+evaluationDependsOn("launcher")
+
+dependencies {
+    compileOnly project("common")
+}
+
+FileCollection launcherTar = project("launcher").tasks.named("tar").get().outputs.files
+FileCollection commonJar = project("common").tasks.named("jar").get().outputs.files
+
+Provider<Copy> copyLauncher = tasks.register("copyLauncher", Copy) {
+    from(launcherTar)
+    into(new File(project.buildDir, "resources/main/META-INF"))
+    rename 'launcher-.*\\.tar', 'flask-launcher.tar'
+}
+
+tasks.named("processResources") {
+    inputs.files(copyLauncher.get().outputs)
+}
+
+jar {
+    manifest {
+        attributes "version" : archiveVersion.get()
+    }
+    from(zipTree(commonJar.singleFile))
+}
+
+gradlePlugin {
+    plugins {
+        flaskPlugin {
+            id = 'net.corda.plugins.flask'
+            implementationClass = 'net.corda.gradle.flask.FlaskPlugin'
+        }
+    }
+    automatedPublishing = false
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    ['compileOnly', 'annotationProcessor', 'testCompileOnly', 'testAnnotationProcessor'].each { conf ->
+        add(conf, [group: "org.projectlombok", name: "lombok", version: lombok_version])
```

Comment:
no, it's only an annotation processor that changes the compiler generated bytecode, it is not required to be present at runtime
It isn't strictly required either, the only reason I use it when writing Java code is because I find Java checked exceptions unbearable